avaframe/in3Utils/getReleaseAreaCfg.ini

```diff
@@ -0,0 +1,50 @@
+### Config File - This file contains the main settings for the simulation run
+## Set your parameters
+# This file is part of Avaframe.
+
+
+# General Topography parameters ----------------------
+[GENERAL]
+hr = 200.0
+vol = 100000.0
+dh = 1.0
+xStart = 100.
+lenP = 4
+showplot = True
+outputtxt = True
+
+# hr        release are vertical stretch
+# vol       volume of snow in release area
+# dh        release snow thickness
+# xStart    upper margin of release area, x-coordinate
+# lenP      number of polygon points
+# showplot  show plot of release area
+# outputtxt copy the output to txt file to be read in QGIS
```

Comment:
Make it consistent with other ini files and put the comments above each line...

Reply:
adjusted